src/bootstrap/src/core/config/config.rs

```diff
@@ -2871,14 +2871,7 @@ impl Config {
 
         // Warn if there were changes to the compiler or standard library since the ancestor commit.
         let mut git = helpers::git(Some(&self.src));
-        git.args(["diff-index", "--quiet", &commit, "--"]);
-
-        // Handle running from a directory other than the top level
-        let top_level = &self.src;
-
-        for path in modified_paths {
-            git.arg(top_level.join(path));
-        }
+        git.args(["diff-index", "--quiet", &commit, "--"]).args(modified_paths);
```

Comment:
Question: I'm not super familiar with git specifics, but could this ever be a problem if say, someone tries to run `./x whatever` from *another* directory that is also a git repo?

Reply:
We already set the working dir for the git command, so it shouldn't matter.